### PR TITLE
update peer deps

### PR DESCRIPTION
**Pull request by jorgecasar**
Allow also @astrojs/db 0.13.x, that was released 3 days ago

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -37,7 +36,7 @@
 		"typescript": "^5.5.3"
 	},
 	"peerDependencies": {
-		"@astrojs/db": "^0.12.0",
+		"@astrojs/db": "0.12 || 0.13",
```
Comment (pilcrowonpaper) [inline package.json]:
```suggestion
		"@astrojs/db": ">= 0.12",
```

Comment (jorgecasar) [reply package.json]: I was thinking on it and finally decide the limited version, but I also prefer keep it open to future version

**Comment by millette** [comment]
Now at v0.14

**Comment by jorgecasar** [comment]
@pilcrowOnPaper please, merge this PR to be able to use your adaptar into version greater or equal than 0.12. Thanks!

**Comment by millette** [comment]
@jorgecasar Meanwhile, I'm using my own fork: <https://www.npmjs.com/package/@waglo/lucia-adapter-astrodb>

**Comment by pilcrowonpaper** [comment]
Sorry, totally missed @jorgecasar's push.

**Comment by pilcrowonpaper** [comment]
Thanks for the PR!

**Pull request merged**